modules/carousel/jetpack-carousel.js

```diff
@@ -445,6 +445,11 @@ jQuery(document).ready(function($) {
 				return;
 			}
 
+			// skip if the parent is not a link
```

Comment:
Yes and no 😄 . The container is actually the [parent](https://github.com/Automattic/jetpack/blob/master/modules/carousel/jetpack-carousel.js#L441), but I will change this if you want.